js/service.js

```diff
@@ -1,11 +1,11 @@
 import env from "../config/env.generated.json";
+import BluebirdPromise from "bluebird";
```

Comment:
I was looking for a way to abort a Promise so that on the search page previous search result won't accidentally interfere later searches. `bluebird` has this feature handy so I used it. See `fetchData()` method in `project-loader.jsx`.